README.md

```diff
@@ -11,17 +11,17 @@ Follow below steps to get started with Tezster-CLI.
 ## Prerequisites
 
 1. Node v. 12.x+
-2. Install Docker
+2. Docker v. 2.3.0.2+
 
 ### OS Support
 1. Linux (Ubuntu and Debian)
 2. Mac OS X
 3. Windows 10 <br />
 
-*Note : Currently we are supporting Linux platform for running local nodes. You may encounter local node related issues in Mac OS X and Windows due to docker. If you are on Mac OS X and Windows, you can switch to remote nodes to use our functionalities.*
+*Note: If you are on **windows**, we would recommend to use powershell as administrator. If you want to use **windows subsystem for winux (WSL)** on windows, please do follow instructions for initial setup as explained in [section](https://docs.tezster.tech/tezster-cli#windows-subsystem-for-linux-wsl).*
```

Comment:
- replace 'section' with actual section name
- please follow the instrusctions. remove 'do'